library/telegraf

```diff
@@ -2,29 +2,26 @@ Maintainers: Josh Powers <jpowers@influxdata.com> (@powersj),
              Sven Rebhan <srebhan@influxdata.com> (@srebhan),
              Dane Strandboge <dstrandboge@influxdata.com> (@DStrand1)
 GitRepo: https://github.com/influxdata/influxdata-docker.git
-GitCommit: 13c39ad36b360ff6c48284d1fd296239f4fc6e4e
+GitCommit: c0beea141b6a373b9d3feb2b1b7c99f20542b707
 
 
-Tags: 1.28, 1.28.5
-Architectures: amd64, arm32v7, arm64v8
-Directory: telegraf/1.28
-
-Tags: 1.28-alpine, 1.28.5-alpine
-Architectures: amd64, arm64v8
-Directory: telegraf/1.28/alpine
-
 Tags: 1.29, 1.29.5
 Architectures: amd64, arm32v7, arm64v8
 Directory: telegraf/1.29
 
 Tags: 1.29-alpine, 1.29.5-alpine
-Architectures: amd64, arm64v8
```

Comment:
Is the removal of `arm64v8` here and on `1.30-alpine` intentional?

Reply:
definitely not intentional, pushed a change to re-add